ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureActivity.java

```diff
@@ -36,7 +38,15 @@ public void onValidated(Context context, ValidateResponse validateResponse, Stri
                 handleValidated(validateResponse, s);
             }
         });
-        onCreateInternal(cardinalClient);
+
+        new Handler(Looper.getMainLooper()).post(new Runnable() {
```

Comment:
I'm not sure I fully see the benefit of adding this. We get an error message without it (cca_continue error) vs this error (internal error). It doesn't seem that much more useful to us or the merchant to have that specific error and this adds a delay to every 3DS transaction. What do you think?

Reply:
The delay won't be noticeable this is a way to allow `handleValidated()` a chance to be called.